File: solutions/COVID_19/COVID_19_4.c
// QUESTION URL: https://www.codechef.com/problems/COVID_19
// STATUS: accepted
// TIME: 0.00
// MEMORY: 5.2M

#include <stdio.h>

int main(void) {
    int t, n, m;
    scanf("%d", &t);
    
    while(t--){
        scanf("%d%d", &n, &m);
        printf("%d\n", (n-n/2)*(m%2 + m/2));
    }
    
	return 0;
}

File: solutions/COVID19/COVID19_4.py
# QUESTION URL: https://www.codechef.com/problems/COVID19
# STATUS: wrong answer
# TIME: 0.00
# MEMORY: 0M

for _ in range(int(input())):
    n = int(input())
    x = list(map(int, input().split()))
    lst = []
    last = 0

    for i in range(n - 1):
        if x[i + 1] - x[i] > 2:
            lst.append(x[last: i + 1])
            last = i + 1

    lst.append(x[last:])

    print(len(min(lst)), len(max(lst)))


